src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -695,6 +699,19 @@ void AppHost::_GlobalHotkeyPressed(const long hotkeyIndex)
             args.SummonBehavior().ToggleVisibility(summonArgs.ToggleVisibility());
             args.SummonBehavior().DropdownDuration(summonArgs.DropdownDuration());
 
+            switch (summonArgs.Monitor())
+            {
+            case Settings::Model::MonitorBehavior::Any:

```

Comment:
Annoying to have to do this but I suspect it's just so you don't cross dependencies between remoting and settings model, yeah?

Reply:
it absolutely 100% is. In the future, there will be `TSM::MonitorBehavior::OnCurrent`, which will actually translate to `Remoting...InPlace` as well, so it's not totally 1-1 (though these 3 totally are)

Reply:
Good enough. Thanks.